### DISCUSSION TITLE
Need help in cloning a repo

### DISCUSSION THREAD

**Discussion by Navya-1817** [community/community#127905] [New to GitHub]
Can you please help with cloning a repo? I am getting this error message attached herewith
![IMG20240609195244](https://github.com/community/community/assets/161189318/05677d34-90e4-4afd-a6b3-fc6d7bc81f49)

**Comment by Rishi-Sudhakar** [comment]
<img width="682" alt="Screenshot 2024-06-09 at 9 24 11 PM" src="https://github.com/community/community/assets/79398572/68b9a921-ccf4-4eb3-9c9a-223b3b273531">


I was able to clone perfectly. Your issue might be an unstable internet connection.

**Reply by Navya-1817** [reply]
SSH is way too complicated for me... isn't there anything that could be done to that error?

**Reply by Navya-1817** [reply]
Oh okay

**Comment by ModithaM** [comment]
git config --global http.postBuffer 524288000
Run This Command and Try Again.
or Use Github Desktop Version. Its Easy.

**Reply by Navya-1817** [reply]
![IMG20240609220620](https://github.com/community/community/assets/161189318/aafec3f8-263b-4af5-9bdf-190bae406d6b)
![IMG20240609214127](https://github.com/community/community/assets/161189318/d3327cd3-405a-40d5-b2fd-2a55c76ed47e)
Neither worked
